packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.tsx

```diff
@@ -408,7 +431,7 @@ const FolderDataGrid = ({
                     return (
                         <FormattedMessage
                             id="comet.dam.folderSize"
-                            defaultMessage="{number} {number, plural, one {item} other {items}}"
+                            defaultMessage="{number} {number, plural, one {file} other {files}}"
```

Comment:
Let's keep item here because it's not just files but also folders

```suggestion
                            defaultMessage="{number} {number, plural, one {item} other {items}}"
```